py/stencila/schema/types.py

```diff
@@ -9,9 +9,11 @@
 
 Enum1 = Enum("1", ["ascending", "descending", "unordered"])
 
-Enum2 = Enum("2", ["data", "header"])
+Enum2 = Enum("2", ["string", "int", "float", "boolean", "json"])
 
-Enum3 = Enum("3", ["header", "footer"])
+Enum3 = Enum("3", ["data", "header"])
+
+Enum4 = Enum("4", ["header", "footer"])
```

Comment:
Just a thought, I wonder if you could generate named enums, to avoid these unnecessary diffs e.g.

```python
TableCellKind = Enum("TableCellKind", ["header", "footer"])
```

Reply:
I changed it to generate `Enum<n>` to shorten the line lengths to be PEP8. Since we aren't PEP8 anyway I could look at going with named enums or hashes again.